templates/core/terraform/update_tags.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Not so keen on this file being an exact copy of `devops/terraform/update_tags.sh`, is it possible to just reference one script?

Reply:
It's not an exact copy as it targets a different resource group...
Most folders have a deploy/destroy script that almost identical to each other.
Never the less, I guess I could move this file to devops/scripts and send the full resource id to update. 
Will that be better @tanya-borisova ?

Reply:
Yeah I'm also not keen on the other almost identical scripts but obviously not asking for it to be fixed in this PR. 
Your idea sounds good, IMO that would be better

Reply:
After reconsidering, decided to leave as is